src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitService.java

```diff
@@ -296,4 +370,48 @@ public List<LimitViolationType> getLimitTypes(UUID resultUuid) {
     public List<com.powsybl.shortcircuit.Fault.FaultType> getFaultTypes(UUID resultUuid) {
         return resultService.findFaultTypes(resultUuid);
     }
+
+    public Optional<ShortCircuitParametersInfos> getParameters(final UUID parametersUuid) {
+        return parametersRepository.findById(parametersUuid).map(ShortCircuitService::fromEntity);
+    }
+
+    @Transactional
+    public boolean deleteParameters(final UUID parametersUuid) {
+        final boolean result = parametersRepository.existsById(parametersUuid);
+        if (result) {
+            parametersRepository.deleteById(parametersUuid);
+        }
+        return result;
+    }
+
+    @Transactional
+    public Optional<UUID> duplicateParameters(UUID sourceParametersUuid) {
+        return parametersRepository.findById(sourceParametersUuid)
+                                   .map(entity -> {
+                                       entityManager.detach(entity);
+                                       entity.setId(null);
+                                       return entity;
+                                   })
+                                   .map(parametersRepository::save)
+                                   .map(AnalysisParametersEntity::getId);
```

Comment:
Don't use entityManager detach directly it could have side effects

Reply:
what side effect? I used it to have an initialised entity to not have to implement a function to duplicate it (with the maintenance on long term to not forget a field).